pkg/monitoring/virt-controller/metrics/metrics.go

```diff
@@ -1,18 +1,74 @@
+/*
```

Comment:
@machadovilaca I believe path should be  pkg/monitoring/metrics/virt-controller/metrics.go
For this and the rest of the metrics.

Reply:
I don't think it makes sense, `virt-controller` is an application which has metrics and rules, not the other way around
in pkg/monitoring we are configuring each of the kubevirt apps:
- `virt-controller`
- `virt-api`
- `virt-operator`
...

and each application has its own metrics and rules
- `virt-controller/metrics`
- `virt-controller/rules`
- `virt-api/metrics`
...

Reply:
I disagree. I think we should have `/pkg/monitoring/metrics` and in it we can have the per app separation.

Reply:
I believe that since our team focus is more on standardizing and managing monitoring across applications my suggestion is more suitable. We are less application-centric in this aspect, but we can still have the app separation under each of the monitoring components, metrics/recordingrules/alertrules.
It would be easier for us to focus on monitoring (e.g., creating or updating metrics) to find all relevant components in one place, When updating monitoring patterns or rules, it's simpler to navigate and make changes across all applications and it encourages a uniform approach to monitoring across the different applications.

Reply:
but all the components won't be in the same place anyway they will still be separated... but we will be going against the intuitive approach, which is going from the bigger concept to the small

anyway this PR does not introduce the 'pkg/monitoring/virt-controller/metrics'  package which was added in https://github.com/kubevirt/kubevirt/commit/e0453d1a581fd568ee1c40966f900e8b3802384f

so this discussion should be in a separate issue

Reply:
We should be looking at the monitoring package in monitoring approach. I believe it makes more sense to have the first logic separation as:
/monitoring
  /metrics
  /alerts
  /recordingrules
  /tools

Reply:
@machadovilaca As agreed, please update the monitoring pkg as mentioned above. Thanks.